### ISSUE TITLE
[auto-generated]

### ISSUE THREAD

**Issue by goro341** [KronicDeth/intellij-elixir#1673]
# Exception

## Stacktrace

```
com.intellij.diagnostic.PluginException: Language: EEx stub version should call super.getStubVersion() [Plugin: org.elixir_lang]
	at com.intellij.ide.plugins.PluginManagerCore.createPluginException(PluginManagerCore.java:405)
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:12)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:58)
	at com.intellij.diagnostic.PluginException.logPluginError(PluginException.java:75)
	at com.intellij.psi.stubs.StubVersionMap.version(StubVersionMap.java:223)
	at com.intellij.psi.stubs.StubVersionMap.typeAndVersion(StubVersionMap.java:207)
	at com.intellij.psi.stubs.StubVersionMap.updateState(StubVersionMap.java:97)
	at com.intellij.psi.stubs.StubVersionMap.<init>(StubVersionMap.java:63)
	at com.intellij.psi.stubs.StubUpdatingIndex$MyIndex.<init>(StubUpdatingIndex.java:263)
	at com.intellij.psi.stubs.StubUpdatingIndex.createIndexImplementation(StubUpdatingIndex.java:258)
	at com.intellij.util.indexing.FileBasedIndexImpl.createIndex(FileBasedIndexImpl.java:493)
	at com.intellij.util.indexing.FileBasedIndexImpl.initIndexStorage(FileBasedIndexImpl.java:409)
	at com.intellij.util.indexing.FileBasedIndexImpl.registerIndexer(FileBasedIndexImpl.java:369)
	at com.intellij.util.indexing.FileBasedIndexImpl.access$5300(FileBasedIndexImpl.java:107)
	at com.intellij.util.indexing.FileBasedIndexImpl$FileIndexDataInitialization.lambda$initAssociatedDataForExtensions$0(FileBasedIndexImpl.java:2448)
	at com.intellij.util.indexing.IndexInfrastructure$DataInitialization.executeNestedInitializationTask(IndexInfrastructure.java:173)
	at com.intellij.util.indexing.IndexInfrastructure$DataInitialization.runParallelNestedInitializationTasks(IndexInfrastructure.java:161)
	at com.intellij.util.indexing.IndexInfrastructure$DataInitialization.call(IndexInfrastructure.java:122)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:222)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:30)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:201)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:221)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:190)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by KronicDeth** [comment]
This was fixed in #1124, which was first released in [v7.5.0](https://github.com/KronicDeth/intellij-elixir/releases/tag/v7.5.0) on June 34d, 2018.  I'm not sure why you would see the error on November 11th, 2019.  Are you using a really old version of an IDE and so can't get updates?

**Comment by KronicDeth** [comment]
Duplicate of #1619.
